build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
Can't use a custom build script on flathub. Do you actually need a version for each driver?